data/strings/types_strings.txt

```diff
@@ -11054,6 +11136,80 @@
     zh-Hans = 塑像
     zh-Hant = 塑像
 
+  [type.historic.memorial.stolperstein]
+    en = Stolperstein
+    ar = حجر عثرة
+    be = Камяні спатыкнення
+    bg = Cпънка
+    cs = Stolperstein
+    da = Stolpersten
+    de = Stolperstein
+    el = Λίθοι Μνήμης
+    es = Stolperstein
+    eu = Stolperstein
+    fa = سنگ مانع
+    fi = Kompastuskivet
+    fr = Stolperstein
+    he = אבן נגף
+    hu = Botlatókő
+    id = Stolperstein
+    it = Pietre d'inciampo
+    ja = ストルパーシュタイン
+    ko = 장애물
+    mr = अडथळा
+    nb = Snublesteiner
+    nl = Stolperstein
+    pl = Stolperstein
```

Comment:
Literally `kamień pamięci` in Polish but I suspect `Stolperstein` is used for this.